addons/web/static/src/js/fields/field_wrapper.js

```diff
@@ -111,14 +116,27 @@ odoo.define('web.FieldWrapper', function (require) {
         isValid() {
             return this.componentRef.comp.isValid;
         }
+        removeInvalidClass() {
+            return this.componentRef.comp.removeInvalidClass(...arguments);
+        }
         reset(record, event) {
             return this.update({record, event});
         }
-        setIDForLabel(id) {
-            return this.componentRef.comp.setIDForLabel(id);
+        setIDForLabel() {
+            return this.componentRef.comp.setIdForLabel(...arguments);
+        }
+        setInvalidClass() {
+            return this.componentRef.comp.setInvalidClass(...arguments);
         }
         updateModifiersValue(modifiers) {
-            return this.componentRef.comp.updateModifiersValue(modifiers);
+            if (this.props.options.attrs) {
+                Object.assign(this.props.options.attrs, modifiers);
```

Comment:
I recall that modifiers can be defined at multiple spots:
- at their own attribute on the node: `<field readonly="" />`
- in the `attrs` attribute of the node:  `<field attrs="{'readonly': domain}" />`